std/range/interfaces.d

```diff
@@ -208,6 +208,13 @@ interface RandomAccessInfinite(E) : ForwardRange!E {
 interface InputAssignable(E) : InputRange!E {
     ///
     @property void front(E newVal);
+
+    @property E front();
+}
+
```

Comment:
That would be inconsistent. Other similar unittests such as https://github.com/dlang/phobos/blob/master/std/range/interfaces.d#L244 aren't documented.